enterprise/models.py

```diff
@@ -915,10 +915,22 @@ def get_paginated_content(self, query_parameters):
         Returns:
             dict: The paginated discovery service search results.
         """
+        results = []
         query = self.content_filter.copy()
         query.update(query_parameters)
         catalog_client = CourseCatalogApiServiceClient(self.enterprise_customer.site)
-        return catalog_client.get_search_results(query, traverse_pagination=False)
+        search_results = catalog_client.get_search_results(query, traverse_pagination=False)
+        for content in search_results['results']:
```

Comment:
Save yourself a variable.
```
for content in catalog_client.get_search_results(query, traverse_pagination=False):
```

Reply:
You don't need the results variable.
```
search_results = []
for content in catalog_client.get_search_results(query, traverse_pagination=False):
```
And then append valid content items to the `search_results` list.

Reply:
@douglashall I am using that variable [HERE](https://github.com/edx/edx-enterprise/pull/229/files#diff-4bcae564bc272f5c2e6753dc6021eb1cR931). Is that variable making the statement complex?

Reply:
Oh, nevermind. I see why you need it now. Thanks.